### PR TITLE
Wip

### PR DESCRIPTION
**Pull request by vicfriedman**
addresses issue #2 @jmburges @AnnJohn

### CODE REVIEW

**Comment by AnnJohn** [comment]
Reassigned to @ipc103

File: README.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
 # JavaScript Math
 
-## Overview
+In Ruby solving for the square root of a number involves using a class method `.sqrt` on the `Math` class. While JavaScript supports a lot of the same mathematical operations as Ruby, there is one big difference. `Math` in JavaScript is actually a single built-in object that you can perform operations on. This `Math` object does have plenty of convenience methods that allow you to do math simply and easily. You can read more about math in JavaScript [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math).
```
Comment (ipc103) [inline README.md]: I'm a little bit confused here- what's the difference between the `Math` class in Ruby and the `Math` object in Javascript?

File: README.md [hunk 2]
```diff
@@ -76,25 +75,24 @@ Math.sqrt(0);
 
 ## Order of Operations
 
-Just like in Ruby, parentheses indicate precedence. For instance, the equation to convert Fahrenheit to Celsius Fahrenheit is to first subtract thirty-two then multiply by five-ninths. To prioritize this subtraction before the multiplication happens, we wrap it in parentheses.
+Just like in Ruby, parentheses indicate precedence. For instance, the equation to convert Fahrenheit to Celsius, Fahrenheit is to first subtract thirty-two then multiply by five-ninths. To prioritize this subtraction before the multiplication happens, we wrap it in parentheses.
```
Comment (ipc103) [inline README.md]: should this be "the equation to convert Fahrenheit to Celsius is to first subtract..."

File: README.md [hunk 3]
```diff
@@ -105,8 +103,9 @@ Something to be aware of as a Rubyist learning JavaScript is that JavaScript wil
 # => TypeError: no implicit conversion of Fixnum into String
 ```
 
-Unlike Ruby, JavaScript will convert this final number into a string and tack that string onto the end:
+Unlike Ruby, JavaScript will convert this final number into a string and tack that string onto the end.
 
+Javascript:
 ```javascript
 // JavaScript
```
Comment (ipc103) [inline README.md]: below, how do we include the Closure library?

**Comment by ipc103** [comment]
finished review - passing back to @vicfriedman

File: README.md (hunk 1 above)
Comment (vicfriedman) [inline README.md]: @ipc103 `Math` in JavaScript is an object itself, which have predefined functions you can call on it (like calling instances methods). `Math` in Ruby is a class that you call class methods on.  

Does that make more sense?

File: README.md (hunk 3 above)
Comment (vicfriedman) [inline README.md]: @ipc I was wondering if it was even worth it to include that part of this reading

File: README.md (hunk 2 above)
Comment (vicfriedman) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (ipc103) [inline README.md]: yes sounds great!

File: README.md (hunk 3 above)
Comment (ipc103) [inline README.md]: yea i think it's confusing here - maybe put it in a later readme if math with money comes up?

Comment (vicfriedman) [inline README.md]: @ipc103 removed!

**Comment by vicfriedman** [comment]
finished updating - passing back to @ipc103 before it goes to @jmburges (@AnnJohn)

**Comment by vicfriedman** [comment]
actually only minor changes - sending to @jmburges

**Pull request merged**